src/operators.jl

```diff
@@ -382,6 +397,11 @@ function _multiply!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs
     ret
 end
 
+# this computes lhs.'*rhs and places it in ret
+function _multiplyt!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs::SparseMatrixCSC, rhs::Array)
+    _multiply!(ret, transpose(lhs), rhs) # for sparse matrices this should be cheap
```

Comment:
I'm not sure if i'd call transposing a sparse matrix cheap, it actually could be relatively slow. Maybe leave a smarter implementation as a TODO?